inc/commonitilobject.class.php

```diff
@@ -470,27 +470,29 @@ function getDefaultActorRightSearch($type) {
 
 
    /**
-    * Count active ITIL Objects requested by a user
+    * Count active ITIL Objects
     *
-    * @since 0.83
+    * @since 9.3.1
     *
-    * @param $users_id integer ID of the User
+    * @param object  $linkclass Link class instance
```

Comment:
Maybe we could also accept the name of the class (example: `User::class`), and create the instance if $linkclass is a `string`.
```
if (is_string($linkclass) && class_exists($linkclass)) {
   linkclass = new linkclass();
}
if (!($linkclass instanceof CommonDBTM)) {
   throw new \InvalidArgumentException();
}
```
If we decide to not accept the class name as a string, we should add the prototype to parameter to prevent call on undefined `getTable()` or `getForeignKeyField()` methods:
```
private function countActiveObjectsFor(CommonDBTM $linkclass, $id, $role) {
```